src/rust/src/backend/ciphers.rs

```diff
@@ -358,8 +364,8 @@ impl PyAEADEncryptionContext {
         let ctx = get_mut_ctx(self.ctx.as_mut())?;
         let result = ctx.finalize(py)?;
 
-        // XXX: do not hard code 16
-        let tag = pyo3::types::PyBytes::new_with(py, 16, |t| {
+        // Allocate buffer for GCM tag
+        let tag = pyo3::types::PyBytes::new_with(py, GCM_STANDARD_TAG_SIZE, |t| {
```

Comment:
Sorry, but this isn't actually addressing the comment: the comment is saying that the `PyAEADEncryptionContext` should work for other tag sizes.